Library/Formula/dcd.rb

```diff
@@ -0,0 +1,13 @@
+require "formula"
+
+class Dcd < Formula
+  homepage "https://github.com/Hackerpilot/DCD"
+  url "https://github.com/Hackerpilot/DCD.git", :using => :git, :tag => "v0.5.1"
```

Comment:
there are submodules.

Why is ```:revision``` necessary as well? The tag specifies everything needed, no?